File: 633-sum-of-square-numbers/sum-of-square-numbers.java
class Solution {
    public boolean judgeSquareSum(int c) {
        for(int div=2; div*div<=c; div++){
            if(c%div==0){
                int ec=0;
                while(c%div==0){
                    ec++;
                    c/=div;
                }
                if(div%4==3 && ec%2!=0) return false;
            }
        }
        return c%4!=3;
    }
}

File: 633-sum-of-square-numbers/sum-of-square-numbers.py
class Solution:
    def judgeSquareSum(self, c: int) -> bool:
        div=2
        while div*div<=c:
            if c%div==0:
                ec=0
                while c%div==0:
                    ec+=1
                    c//=div
                if div%4==3 and ec%2!=0:    return False
            div+=1
        return c%4!=3